### PR TITLE
"CAMEL:17092:Debezium 1.7.0 - Oracle Connector"

### PR DESCRIPTION
**Pull request by ramu11**
"CAMEL:17092:Debezium 1.7.0 - Oracle Connector" 

Added camel-debezium-oracle connector

### CODE REVIEW

File: components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
+= Debezium Oracle Connector Component
+:doctitle: Debezium Oracle Connector
+:shortname: debezium-oracle
+:artifactid: camel-debezium-oracle
+:description: Capture changes from a Oracle database.
+:since: 3.0
```
Comment (davsclaus) [inline components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc:6]: 3.17

File: components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+include::partial$component-configure-options.adoc[]
+include::partial$component-endpoint-options.adoc[]
+// component options: END
+
+// endpoint options: START
+
+// endpoint options: END
+
+For more information about configuration:
+https://debezium.io/documentation/reference/0.10/operations/embedded.html#engine-properties[https://debezium.io/documentation/reference/0.10/operations/embedded.html#engine-properties]
```
Comment (davsclaus) [inline components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc:60]: 0.10 -> 1.8 as its based on debezium 1.8.x

File: components/camel-debezium/camel-debezium-oracle/src/main/java/org/apache/camel/component/debezium/DebeziumOracleEndpoint.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.apache.camel.Category;
+import org.apache.camel.component.debezium.configuration.OracleConnectorEmbeddedDebeziumConfiguration;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriParam;
+
+/**
+ * Capture changes from a Oracle database.
+ */
+@UriEndpoint(firstVersion = "3.0.0", scheme = "debezium-oracle", title = "Debezium Oracle Connector",
```
Comment (davsclaus) [inline components/camel-debezium/camel-debezium-oracle/src/main/java/org/apache/camel/component/debezium/DebeziumOracleEndpoint.java:28]: 3.0.0 -> 3.17.0

File: components/camel-debezium/pom.xml [hunk 4]
```diff
@@ -38,6 +38,7 @@
         <module>camel-debezium-mysql</module>
         <module>camel-debezium-postgres</module>
         <module>camel-debezium-sqlserver</module>
+        <module>camel-debezium-oracle</module>
```
Comment (davsclaus) [inline components/camel-debezium/pom.xml:41]: Can you put this in a sorted A..Z position

Comment (ramu11) [reply components/camel-debezium/pom.xml]: done

File: components/camel-debezium/camel-debezium-oracle/src/main/java/org/apache/camel/component/debezium/DebeziumOracleEndpoint.java (hunk 3 above)
Comment (ramu11) [reply components/camel-debezium/camel-debezium-oracle/src/main/java/org/apache/camel/component/debezium/DebeziumOracleEndpoint.java]: done

File: components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc (hunk 2 above)
Comment (ramu11) [reply components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc]: done

File: components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc (hunk 1 above)
Comment (ramu11) [reply components/camel-debezium/camel-debezium-oracle/src/main/docs/debezium-oracle-component.adoc]: done

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Pull request merged**